oem-reownership/oem.example

```diff
@@ -0,0 +1,21 @@
+#OEM name shown on onboarding screen if found under /boot/oem when OEM Reownership Wizard starts
+oem_name=Insurgo Open Technologies
+#
+#The following are states being checked against prior to entering that specific stage. 
+# Once a stage is done, it is appended under /boot/oem
+#
+#If not found in file, shows Onboarding first message containing oem_name above if defined, else "your OEM" is shown
+onboarding
+#When not present in this file, reencrypts LUKS container with oem_luks_actual_Disk_Recovery_Key defined under /media/oem-provisioning
+luks_reencrypted
+#When not present in this file, changes oem_luks_actual_Disk_Recovery_Key with oem_luks_new_Disk_Recovery_Key defined under /media/oem-provisioning
+luks_passwd_changed
+#When not present in this file, factory resets Librem Key/NitroKey with: oem_gpg_Admin_PIN, oem_gpg_User_PIN, oem_gpg_real_name, oem_gpg_email, oem_gpg_comment, defined under /media/oem-provisioning.
+#The wizard generates new keypairs, injects the public key and trust database into the ROM and reboots. 
+#Upon reboot, the oem_gpg_Admin_PIN is used to automatically seal HOTP secret into the Librem Key/NitroKey
+gpg_factory_resetted
+#When not present in this file, reown the TPM with oem_TPM_Owner_Password, which is recommended to be the same as oem_gpg_Admin_PIN
+tpm_reowned
+#Once all stages are done, the Wizard removes /boot/oem, and ensures that /boot file checksum file is generated and signed with oem_gpg_User_PIN.
+#
+#Then the wizard finally launches the selection of a new boot default, and asks the user if he wants to modifie the disk and if he wants to define a Disk Unlock Key. In such case, oem_TPM_Owner_Password is used to seal a new TOTP secret, oem_gpg_Admin_PIN is used to seal a new HOTP secret, and oem_gpg_User_PIN is used to sign configuration files checksums that changed under /boot. 
```

Comment:
s/modifie/modify (doesn't look bad to me as long as it doesn't break anything (having unencrypted rootfs or password encrypted lvm)) but unfortunately very little time to review right now -.-)

Reply:
@merge : didn't tought of unencrypted rootfs at all and it's not currently supported with actual modifications. Will have to review code for that, while..... Does it make sense to use Heads to protect hardware for which data is not protected at all? There is a use case for that? Heads offers integrity protection for /boot considering rootfs should be encrypted IMHO.

Let me know if you see a requirement for this.

Reply:
Yes it makes sense for Heads to protect hardware that doesn't have an encrypted /. Heads doesn't claim to protect / (by default) anyway and more importantly, Heads *can't* protect /. It claims to protect the boot process up to and including loading and executing the kernel. The underlying OS needs to take over at that point with the firm foundation that everything that executed before it is now trusted and not tampered with.

We need to be careful to not have Heads dictate security policy outside of Heads to users.

Reply:
@kylerankin @merge   Please review [this](https://github.com/osresearch/heads/pull/551/commits/677f81047551e6ad72333d436dca560b5550b9f2). 
Basically, that skips kexec-save-key code part to go directly [here](https://github.com/osresearch/heads/blob/677f81047551e6ad72333d436dca560b5550b9f2/initrd/bin/kexec-save-default#L187) and continue execution.

Reply:
thanks! that's very imortant imo. 1. like kyle said, it must totatlly be up to the runtime kernel and user, what rootfs to mount, and 2. encryption and tamper-protection are fundamentally different things. it totally makes sense to have an unencrypted rootfs, with (maybe) verification/tamper-protection in place (way outside of Heads' scope).